app/models/concerns/conclusion_reviews/gal_pdf.rb

```diff
@@ -66,17 +66,26 @@ def put_gal_cover_on pdf
     end
 
     def put_executive_summary_on pdf, organization
-      title           = I18n.t 'conclusion_review.executive_summary.title'
-      use_alt_project = review.business_unit_type.independent_identification
-      project_label   = use_alt_project ? 'project_alt' : 'project'
-      project_title   = I18n.t "conclusion_review.executive_summary.#{project_label}"
-      project         = review.plan_item.project
+      title                      = I18n.t 'conclusion_review.executive_summary.title'
+      review_key                 = I18n.t "conclusion_review.executive_summary.keywords.review"
+      params                     = { "#{review_key}": review.identification }
+      exec_summary_intro         = review.business_unit_type.exec_summary_intro
+      independent_identification = review.business_unit_type.independent_identification
+      project                    = review.plan_item.project
+
+      full_exec_summary_intro = if exec_summary_intro
```

Comment:
```suggestion
      full_exec_summary_intro = if exec_summary_intro.present?
```

Si lo guardas sin dato, se almacena "", para el if eso es verdadero y siempre entra en el if